### ISSUE TITLE
error rendering request parameter object in large service

### ISSUE THREAD

**Issue by tomaskodaj** [springdoc/springdoc-openapi#2538]
I you have complex objects in request parameters (e.g. query parameter) referenced as "$ref", only first 11 "paths" are rendered correctly as "textarea", rest of them wont be rendered with example, and wont render textarea when you hit "try it out".

This bug was introduced in version 2.4.0... version 2.3.0 was OK.

Using with spring boot 3.2.3.
```
    "paths": {
...
                "parameters": [
                    {
                        "required": true,
                        "examples": {
                            "exampleFullSomeComplexObjectSchema": {
                                "$ref": "#/components/examples/exampleFullSomeComplexObjectSchema"
                            },
                            "exampleMinimalSomeComplexObjectSchema": {
                                "$ref": "#/components/examples/exampleMinimalSomeComplexObjectSchema"
                            }
                        },
                        "schema": {
                            "$ref": "#/components/schemas/SomeComplexObjectSchema"
                        },
                        "description": "Some Complex Object Schema",
                        "in": "query",
                        "name": "someComplexObjectSchema"
                    }
```

**Comment by codespearhead** [comment]
We need an [MRE](https://en.wikipedia.org/wiki/Minimal_reproducible_example#:~:text=In%20computing%2C%20a%20minimal%20reproducible,to%20be%20demonstrated%20and%20reproduced.), like [this one](https://github.com/springdoc/springdoc-openapi/issues/2442#issuecomment-2007786423).

**Comment by tomaskodaj** [comment]
minimal example: [bug-example.zip](https://github.com/springdoc/springdoc-openapi/files/14668420/bug-example.zip)

see last cca 40 items and request parameter "is"... compare to first 60 items... then recompile with 2.3.0 and it is correct... with 2.4.0 it is not... exactly as told in original issue.

**Comment by tomaskodaj** [comment]
Error is in swagger-ui-version = 5.11.8

Please upgrade to 5.11.10, where it seems fixed

**Comment by chaouki-reply** [comment]
We have faced the same behavior but hopefully, this will be fixed when merging [PR#2542](https://github.com/springdoc/springdoc-openapi/pull/2542).
